<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta name="generator" content="HTML Tidy for HTML5 (experimental) for Windows https://github.com/w3c/tidy-html5/tree/c63cc39" />

    <title>RiboVision: Ribosome Visualization Suite.</title>
    <link rel="stylesheet" type="text/css" href="css/style.css" />
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js">
</script>
    <script type="text/javascript" src="js/template.js">
</script>
</head>

<body lang="EN-US" xml:lang="EN-US">
    <div id="HeaderDiv"></div>

    <div id="MenuDiv"></div>

    <div id="ContentDiv">
        <h1>DataSets</h1>

        <p>rRNA secondary structures allow organization and illustration of three-dimensional architecture, local molecular interactions, phylogeny,
		evolution, and function. Our goal is to represent secondary  structures that are as accurate and useful as possible, using 3D crystal structures of
		ribosomes as input.  We offer secondary structures for 23S/28S/5S and 16S/18S rRNAs of <i>E. coli</i>, <i>Thermus thermophilus</i>, <i>Haloarcula marismortui</i>,
        and <i>Saccharomyces cerevisiae</i>. Both the large and small subunit are available for each species, except the small subunit for
        <i>Haloarcula marismortui</i>.  There are two kinds of secondary structures for each: A classic phylogeny-based structure, and a new structure developed
		in house, which is more accurate to the 3D structure. Additionally, we provide high-resolution editable versions of those structures mapped with a variety of data related 
		to molecular interactions and geometry, phylogeny and evolution, and partitioning of rRNA into helices and domains
		(<a href="http://apollo.chemistry.gatech.edu/RibosomeGallery" target="_blank">http://apollo.chemistry.gatech.edu/RibosomeGallery</a>). </p>
		<p>NEW. We are releasing two new structures, <i>Drosophilia melanogaster</i> and <i>Homo sapiens</i>, to represent metazoans. We have decided to provide only the new 3D-based
		secondary structures for metazoans. In the classic format, metazoan secondary structures can not be printed to a single page legibly.  These structures are atomic models, 
		modeled into cyro-EM electron density. </p>
		
        <h1>Scientific Credits</h1>
		<h2>Credits for the new 3D-based Secondary Structures</h2>
		<p>
		Petrov et al., (2013) "Secondary Structure and Domain Architecture of the 23S rRNA", <i>Nucleic Acids Res.</i> doi: 
			10.1093/nar/gkt513
		</p>
        <h2>Credits for the Secondary Structures</h2>

        <p><a href="http://www.rna.icmb.utexas.edu/CAR/1A/" target="_blank">http://www.rna.icmb.utexas.edu/CAR/1A/</a></br></br>
        <a href="http://rna.ucsc.edu/rnacenter/ribosome_images.html" target="_blank">http://rna.ucsc.edu/rnacenter/ribosome_images.html</a></p>

        <p>Woese C R, Magrum L J, Gupta R, Siegel R B, Stahl D A, Kop J, Crawford N, Brosius J, Gutell R, Hogan J J and Noller H F 1980 <i>Nucleic Acids
        Res.</i> <b>8</b> 2275</br></br>
        Noller H F, Kop J, Wheaton V, Brosius J, Gutell R R, Kopylov A M, Dohme F, Herr W, Stahl D A, Gupta R and Woese C R 1981 <i>Nucleic Acids Res.</i>
        <b>9</b> 6167</br></br>
        Noller H F 1984 <i>Annual Review of Biochemistry</i> <b>53</b> 119</br></br>
        Gutell RR, Weiser B, Woese CR, and Noller HF 1985 <i>Prog. Nucleic Acids Res. Mol. Biol.</i> <b>32</b> 155</br></br>
        Gutell R R and Fox G E 1988 <i>Nucleic Acids Research</i> <b>16</b> R175</br></br>
        Gutell R R, Larsen N and Woese C R 1994 <i>Microbiol Rev</i> <b>58</b> 10</br></br>
        Cannone J J, Subramanian S, Schnare M N, Collett J R, D&#39;Souza L M, Du Y, Feng B, Lin N, Madabusi L V, Muller K M, Pande N, Shang Z, Yu N and Gutell
        R R 2002 <i>BMC Bioinformatics</i> <b>3</b> 2</br></br>
        Ben-Shem A, Jenner L, Yusupova G and Yusupov M 2010 <i>Science</i> <b>330</b> 1203</br></br>
		Anger, A. M., et al. (2013). "Structures of the human and Drosophila 80S ribosome." Nature 497(7447): 80-85.</br></br>
	
</p>

        <h2>Credits for the X-ray (or Cryo-EM) Structures</h2>

        <p>Yusupov M M, Yusupova G Z, Baucom A, Lieberman K, Earnest T N, Cate J H and Noller H F 2001 <i>Science</i> <b>292</b> 883</br></br>
        Ban N, Nissen P, Hansen J, Moore P B and Steitz T A 2000 <i>Science</i> <b>289</b> 905</br></br>
        Dunkle J A, Wang L Y, Feldman M B, Pulk A, Chen V B, Kapral G J, Noeske J, Richardson J S, Blanchard S C and Cate J H D 2011 <i>Science</i> <b>332</b>
        981</br></br>
        Ben-Shem A, de Loubresse N G, Melnikov S, Jenner L, Yusupova G and Yusupov M 2011 <i>Science</i> <b>334</b> 1524</br></br></p>
		
		 <h2>Credits to SILVA for Sequences and Alignment</h2>
		<p>
		Pruesse E, Peplies J and Glockner F O 2012 <i>Bioinformatics</i> <b>28</b> 1823</br></br>
		</p>
		<h2>Credits to FR3D for Interaction Data</h2>
		<p>
		Sarver M, Zirbel C L, Stombaugh J, Mokdad A and Leontis N B 2008 <i>J Math Biol</i> <b>56</b> 215</br></br>
		Anger, A. M., et al. (2013). "Structures of the human and Drosophila 80S ribosome." Nature 497(7447): 80-85.</br></br>
		</p>
    </div>

    <div id="FooterDiv"></div>
</body>
</html>
